GVRf/Framework/framework/src/main/java/org/gearvrf/GVRShaderTemplate.java

```diff
@@ -811,7 +810,8 @@ private String makeVertexCopy(String descriptor, String inBase, String outBase)
         }
         return desc;
     }
-    
+
+    protected boolean mWriteShadersToDisk = true;
```

Comment:
Since you are still working on this pull request, please set mWriteShadersToDisk to false.